README.md

```diff
@@ -179,7 +179,7 @@ Bundle size
   </tr>
 </table>
 
-**note**: version 5 is released with version 4 tfor experimental use. Based on it's demand, it'll be developed and the features can be different in final release.
+**note**: version 6 is released with version 5 for experimental use. Based on it's demand, it'll be developed and the features can be different in final release.
```

Comment:
It should be "v6 is released with v4 for experimental use. Based on it's demand, it'll be developed and the features can be different in final release. v5 has same functionality as v4"

Reply:
I integrated the requested change in 13cd673